api/SubtleCrypto.json

```
@@ -1158,6 +1158,170 @@
             "deprecated": false
           }
         },
+        "ECDH": {
+          "__compat": {
+            "description": "`ECDH` algorithm",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/SubtleCrypto/importKey",
+            "spec_url": "https://w3c.github.io/webcrypto/#ecdh",
+            "support": {
+              "bun": {
+                "version_added": false
+              },
+              "chrome": {
+                "version_added": "37"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": "1.18",
+                "partial_implementation": true,
+                "notes": "P-384 supported. Not supported: P-256, P-521."
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "34"
+              },
+              "firefox_android": "mirror",
+              "nodejs": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "7"
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          },
+          "compressed_elliptic_curve_points": {
+            "__compat": {
+              "description": "Key data can contain compressed elliptic curve points.",
```

Optional nit: it's unusual to terminate descriptions with a fullstop, I think?

```suggestion
              "description": "Key data can contain compressed elliptic curve points",
```